test/run-drun-non-ci/ok/oom-query.drun-run.ok

```diff
@@ -1,3 +1,3 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
-Err: IC0503: Canister rwlgt-iiaaa-aaaaa-aaaaa-cai trapped explicitly: RTS error: Cannot grow memory
+Err: IC0502: Canister rwlgt-iiaaa-aaaaa-aaaaa-cai trapped: heap or table out of bounds
```

Comment:
This is still worrying me.

Reply:
Nope this now gives "cycles exhausted". I didn't update it yet.

Reply:
You prolly get it on `master` too.

Reply:
Looks like the replica doesn't give the same cycle budget to queries any more.

Reply:
I am still seeing the problem: 0bd3f620d. I'd like to see the "heap out of bounds", but don't have an idea how to get there.

Reply:
And for me the whole directory seems bit-rotten. It might be my machine but I see (on `master`!)
``` diff
make: Entering directory '/Users/ggreif/motoko/test/run-drun-non-ci'
../run.sh -d *.mo *.drun
inc-oom: [tc] [comp] [valid] [drun-run]
oom-query: [tc] [comp] [valid] [drun-run]
--- oom-query.drun-run (expected)
+++ oom-query.drun-run (actual)
@@ -1,3 +1,3 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
-Err: IC0503: Canister rwlgt-iiaaa-aaaaa-aaaaa-cai trapped explicitly: RTS error: Cannot grow memory
+Err: IC0522: Canister rwlgt-iiaaa-aaaaa-aaaaa-cai exceeded the instruction limit for single message execution.
oom-update: [tc] [comp] [valid] [drun-run]
preupgrade-oom: [tc] [comp] [valid] [drun-run]
upgrade-bug-immut: [tc] [comp] [valid] [drun-run]
--- upgrade-bug-immut.drun-run (expected)
+++ upgrade-bug-immut.drun-run (actual)
@@ -1,4 +1,5 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
 debug.print: pre
-ingress Err: IC0503: Canister rwlgt-iiaaa-aaaaa-aaaaa-cai trapped explicitly: RTS error: Cannot grow memory
+debug.print: post
+ingress Err: IC0503: Canister rwlgt-iiaaa-aaaaa-aaaaa-cai trapped explicitly: deliberate trap
upgrade-bug: [tc] [comp] [valid] [drun-run]
--- upgrade-bug.drun-run (expected)
+++ upgrade-bug.drun-run (actual)
@@ -1,4 +1,5 @@
 ingress Completed: Reply: 0x4449444c016c01b3c4b1f204680100010a00000000000000000101
 ingress Completed: Reply: 0x4449444c0000
 debug.print: pre
-ingress Err: IC0503: Canister rwlgt-iiaaa-aaaaa-aaaaa-cai trapped explicitly: RTS error: Cannot grow memory
+debug.print: post
+ingress Err: IC0503: Canister rwlgt-iiaaa-aaaaa-aaaaa-cai trapped explicitly: deliberate trap
File *.drun does not exist.
Some tests failed:
oom-query.mo upgrade-bug-immut.mo upgrade-bug.mo *.drun
make: *** [Makefile:4: all] Error 1
make: Leaving directory '/Users/ggreif/motoko/test/run-drun-non-ci'
```

Reply:
So this part also needs some TLC in a follow-up!